### DISCUSSION TITLE
Adobe Premiere Pro

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#23] [General]
# Adobe Premiere Pro

Check if Adobe Premiere Pro is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/fac2419f-c62f-4572-85c2-19091d5922ad

**Comment by stbaxter** [comment]
Hi There, Adobe announced yesterday that Premiere Pro is available for CP+ devices.

https://www.windowscentral.com/software-apps/windows-11/windows-11-laptops-with-snapdragon-x-arm-chips-can-now-run-adobe-premiere-pro-under-emulation

**Reply by OpenSource03** [reply]
Updated as per #24
